databricks/koalas/namespace.py

```diff
@@ -199,7 +200,7 @@ def read_csv(
     quotechar=None,
     escapechar=None,
     comment=None,
-    **options
+    **options,
```

Comment:
These `,`s seem to be added by running `dev/reformat`. Is this fine?

Reply:
Nope, it's not supported in Python 3.5.

Reply:
Right, removed!